candy-dispenser/README.md

```diff
@@ -0,0 +1,10 @@
+Python Flask Web Application for controlling a ZmP Candy Dispenser.
+
+## How to use this image
+
+```
+docker run -it --rm --name candy-dispenser --net=host -e HOST=https://mgmt.foundries.io/lshan -e CLIENT=zmp:sn:1d0ca460 hub.foundries.io/candy-dispenser:latest
+
```

Comment:
does it really need --network=host or can we just do "-p 5000:5000" ?

Reply:
We can just use the 5000 port forward, I'll update.